src/autoscaler/metricscollector/collector/apppoller_test.go

```diff
@@ -211,7 +211,7 @@ var _ = Describe("Apppoller", func() {
 					Expect(appid).To(Equal("test-app-id"))
 					Expect(token).To(Equal("bearer test-access-token"))
 
-					if noaa.ContainerEnvelopesCallCount()%2 == 0 {
+					if noaa.ContainerEnvelopesCallCount() >= 2 && noaa.ContainerEnvelopesCallCount() <= 4 {
```

Comment:
@fraenkel Are you then suggesting that we remove this test that we have a fail and retry test below already?